src/screens/FeedScreen/index.js

```diff
@@ -263,7 +262,7 @@ const FeedScreen = (props) => {
         refreshAnonymous={onDeleteBlockedPostCompletedHandle}
         screen="screen_feed"
       />
-    </SafeAreaProvider>
+    </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Same issue as above. Consider reverting back to using `SafeAreaProvider`.

```diff
- </View>
+ </SafeAreaProvider>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->